src/pages/apis/tobacco/event/_fields.yaml

```diff
@@ -0,0 +1,55 @@
+type: object
+properties:
+  report_id:
+    description: "7-digit ICSR number (no spaces)."
+    format: integer
+    is_exact: false
+    possible_values:
+    type: number
+  submission_date:
+    description: "Date of Original Receipt to CTP."
+    format: date
+    is_exact: false
+    possible_values:
+    type: string
+  number_tobacco_products:
+    description: "System-calculated number of Tobacco Product Problems reported, displayed as a whole number, ≥ 0."
+    format: integer
+    is_exact: false
+    type: number
+  number_health_problems:
+    description: "System-calculated number of Health  Problems (i.e., MedDRA terms selected from a standardized list of symptoms, signs, diagnoses and outcomes) reported, displayed as a whole number, ≥0."
+    format: integer
+    is_exact: false
+    type: number
+    possible_values:
+  number_product_problems:
+    description: "System-calculated number of categorical Product Problems  reported, displayed as a whole number, ≥0."
+    format: integer
+    is_exact: false
+    type: number
+    possible_values:
+  nonuser_affected:
+    description: "Displays text reflecting the response to this optional question as “No information provided” if not answered, or Yes/No."
+    format:
+    is_exact: false
+    possible_values:
+    type: boolean
+  product_type:
+    description: "Text reflecting the SRP tobacco Product Type selected by the reporter. If two or more types were reported, both are displayed."
+    format:
+    is_exact: true
+    possible_values:
+    type: string
+  reported_health_problems:
+    description: "Text reflecting the MedDRA terms selected by the reporter.  When two or more terms were reported, they are all displayed and separated by forward slashes. If there are duplicate terms reported such as cough/dry eye, both are displayed. When no health problem is reported the text “No information provided” is displayed."
+    format:
+    is_exact: true
+    possible_values:
+    type: string
+  reported_product_problems:
+    description: "Text reflecting the SRP categorical list of values. When two or more values were reported, they are all displayed and separated by forward slashes. When no product problem is reported the text “No information provided” is displayed."
```

Comment:
Let's take out the line about "When two or more terms" and reformat this to represent it as an array of strings.